core.spec.md

```diff
@@ -418,23 +418,21 @@ This section lays out algorithms for processing core schemas.
 
 Algorithms described in this section may produce *validation failures* if a document does not conform to the requirements core schema document. Validation failures SHOULD halt processing. Some consumers, such as authoring tools, MAY attempt to continue processing in the presence of validation failures, but their behavior in such cases is unspecified.
 
-Algorithms may also produce *warnings* if a document does not conform to requirements. *Warnings* indicate issues with the document which SHOULD be surfaced to users, but which do not necessarily require processing to halt. Consumers SHOULD generally choose to continue processing in the presence of warnings. Some consumers MAY choose to adopt stricter standards, and halt processing on both warnings and failures.
-
 ## Bootstrapping
 
 Determine the name of the core specification within the document.
 
 It is possible to [rename the core feature](#sec-Renaming-core-itself) within a document. This process determines the actual name for the core feature if one is present.
 
 - **Fails** the *Has Schema* validation if there are no SchemaDefinitions in the document
-- **Warns** *Extra Schema* for extra SchemaDefinitions in the document after the first
+- **Fails** *Extra Schema* for extra SchemaDefinitions in the document after the first (note that is is also [invalid GraphQL](http://spec.graphql.org/draft/#sec-Root-Operation-Types)
```

Comment:
Can't we literally get rid of this since it's not a concern of `core` but of valid graphql itself? Like we don't also have to say things like "you can't have duplicate type names" and every other validation that the graphql spec enforces.